libraries/api/discussion_helper.cpp

```diff
@@ -213,7 +303,8 @@ namespace golos { namespace api {
     }
 
     discussion discussion_helper::impl::create_discussion(const comment_object& o) const {
-        return discussion(o, database_);
+        auto x = create_comment_api_object(o);
+        return discussion(x);
```

Comment:
You can refactor ` create_comment_api_object ` to two functions `fill_comment_api_object` and `get_comment_api_object`.
And pass `discussion` to `fill_comment_api_object`.

Reply:
I've added `fill_comment_api_object` method special for `discussion`, that would be enough I guess.

Reply:
So this case is fixed.